cloudferrylib/scheduler/scenario.py

```diff
@@ -68,13 +81,15 @@ def load_scenario(self, path_scenario=None):
             # "rollback" yaml chain can be added to rollback to previous state
             #                                    in case of main chain failure
             self.rollback = migrate.get("rollback")
+        self.load_scenario_with_num_lines(path_scenario)
 
     def get_net(self):
         result = {}
-        for key in ['migration', 'preparation', 'rollback']:
+        for key in STAGES_SCENARIO:
             if hasattr(self, key) and getattr(self, key):
                 scenario = getattr(self, key)
-                checker = ScenarioChecker(self.tasks.keys(), key, scenario)
+                checker = ScenarioChecker(self.tasks.keys(), key, scenario,
+                                          self.raw_scenario_file)
```

Comment:
Why do we need to analyze the raw scenario file? Is the information stored in `self.preparation`, `self.rollback`, `self.namespace` and `self.migration` not enough? 
Please revert the patch to previous version and resolve all comments.